### PR TITLE
Cleanup links

### PR DESCRIPTION
**Pull request by giancarloGiuffra**
- A couple of links have been updated
- Broken links have been removed
- One link was listed 3 times in different topics, I kept it only in on of them
- Formatted the tables

### CODE REVIEW

**Comment by adamjonas** [comment]
Thank you @giancarloGiuffra. Will take a look in the coming days.

File: README.md [hunk 1]
```diff
@@ -14,107 +14,105 @@ There are two portions to this curriculum:
...
...                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (carlaKC) [inline README.md:17]: It's very difficult to see what links have changed here when there's an update to the table structure as well - could you further split out the commits to have the link updates/formatting changes in separate commits?

Comment (giancarloGiuffra) [reply README.md:17]: I split in 2 commits, but still GH diff preview does not catch everything. In Rider it is easier to spot the differences:
image. Hopefully, it helps.

I think this problem is going to happen again every time this list gets updated. I still think, [this](https://github.com/chaincodelabs/lightning-curriculum/pull/11) restructuring would make it easier to update this list in the future.

File: README.md [hunk 2]
```diff
@@ -14,107 +14,105 @@ There are two portions to this curriculum:
...
-What is Lightning?|Lightning ≈ Bitcoin||[Lightning ≈ Bitcoin](https://www.youtube.com/watch?v=8lMLo-7yF5k) |
-What is Lightning?|Lightning ≈ Bitcoin|Revisting tx malleability|[SF Bitcoin Devs Seminar: Transaction Malleability: Threats and Solutions](https://www.youtube.com/watch?v=jyDE-aFqJTs) |
-| How Layers fit together | Overview || [How the layers of Lightning Fit Together Seminar Video](https://youtu.be/krux2v0jt4E) |
-| How Layers fit together | Update || [The Update Layer Seminar Video](https://youtu.be/SoFlRCNdqDg) |
-| How Layers fit together | Transfer || [The Transfer Layer Seminar Video](https://youtu.be/CGE8I8L7BAc) |
-| How Layers fit together | Multihop || [The Multihop Layer Seminar Video](https://youtu.be/P7I-C0_sijg) |
-| How Layers fit together | Others || [The Base an Transport Layers Seminar Video](https://youtu.be/wyri7cc83kQ) |
-|Transfer|Payment channel| General concept|[Payment Channel](https://wiki.ion.radar.tech/tech/lightning/payment-channel) |
```
Comment (carlaKC) [inline README.md:29]: Ideally we'd include something else introducing the general concept of a payment channel? 
Possibly this? https://bitcoinmagazine.com/technical/understanding-the-lightning-network-part-building-a-bidirectional-payment-channel-1464710791

File: README.md (hunk 1 above)
Comment (carlaKC) [reply README.md:17]: > A couple of links have been updated

Could you give me a tl;dr on the links that have changed?

**Comment by adamjonas** [comment]
Sorry for the neglect here

**Pull request merged**